uwblueprint/marillac-place
# Notification refactor

## Notion ticket link
[Refactor notifications service functionality](https://www.notion.so/uwblueprintexecs/Refactor-notifications-service-functionality-854f2a1cb86f431c9ed2f52d323859f7)



## Implementation description
* This update focuses on the notification system, enabling residents and staff to send and receive notifications and announcements. 


## Steps to test
1. Create users. 
2. Utilize Postman to send notifications. Verify that: the intended recipients receive notifications without issues. 
![image](https://github.com/uwblueprint/marillac-place/assets/109609935/51ef620e-090f-479c-818f-238535565592)

Note: We are using a UserStub model, make sure to replace instances of UserStub to User in the actual change.


## What should reviewers focus on?
*  `./graphql/resolvers/notificationResolvers.ts`
*  `./graphql/types/notificationType.ts`
*  `./prisma/schema.prisma`

## Checklist
